src/common/replication_common.cpp

```diff
@@ -41,6 +41,11 @@ DSN_DEFINE_int32("replication",
                  5,
                  "concurrent bulk load downloading replica count");
 
+DSN_DEFINE_uint32("replication",
+                  update_ranger_policy_interval_s,
+                  5,
+                  "every this period(ms) the meta server will do load ranger policy");
```

Comment:
The interval seconds meta server to pull the latest access control policy from Ranger server.

Reply:
done